### ISSUE TITLE
iWork8 Ultimate

### ISSUE THREAD

**Issue by NotKit** [zhaofengli/iwork8#5]
Hello.
I have Cube iWork8 Ultimate (which you referred to as "flashship version"). Here is firmware extracted from Android kernel: https://github.com/onitake/gslx680-acpi/files/193513/cube_iwork8_ultimate.tar.gz

**Comment by zhaofengli** [comment]
:+1: That's good to hear. Have you got it working on Linux?

**Comment by NotKit** [comment]
Yes, got Arch Linux working on it, but suffering from almost the same problems as yours iWork 8. No need for KMS hack though, and now there is dualboot firmware available, which brings proper 64-bit UEFI. Wi-Fi module is the same, For sound NAU8824 codec is used, which I couldn't find any Cherry Trail machine drivers for yet.
